PhysicsTools/Scouting/python/test/run_Run3ScoutingElectronBestTrack_cfg.py

```diff
@@ -0,0 +1,49 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("Demo")
+
+# Load standard configurations
+process.load("FWCore.MessageService.MessageLogger_cfi")
+process.load("Configuration.StandardSequences.GeometryRecoDB_cff")
+process.load("Configuration.StandardSequences.MagneticField_cff")
+process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
+
+# Set the global tag
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, '140X_dataRun3_HLT_v3', '')
+
+# Configure the MessageLogger
+process.MessageLogger.cerr.FwkReport.reportEvery = 10
+
+# Input source
+process.source = cms.Source("PoolSource",
+    fileNames = cms.untracked.vstring(
+        'root://cms-xrd-global.cern.ch///store/data/Run2024I/ScoutingPFRun3/HLTSCOUT/v1/000/386/478/00000/0100d00a-69a6-4710-931f-b1c660f87675.root'  # Replace with your input file
```

Comment:
@smuzaffar, is it possible to copy this file into the cmsbuild user eos area? @arsahasransu would like to use it for a unit test for the new functionality.

Reply:
@mmusich , this file is now available via cms-bot eos area